harvard-lts/jstor_aspace_oai_harvester
# change usage text

**Change usage text.**
* * *

**JIRA Ticket**: [(link)](https://jira.huit.harvard.edu/browse/SS-300)

** Also checking in test cases for invalid ead for https://jira.huit.harvard.edu/browse/SS-80

# What does this Pull Request do?
Replaces usage descripton from example to JSTOR

# How should this be tested?

Check out branch locally
Copy celeryconfig.py.example to celeryconfig.py
Make sure celery installed > pip install celery
Run usage
> python3 ./scripts/run_harvest.py -h
Description should read
`Script for harvesting and processing full JSTOR sets.`

# Test coverage
Yes/No: Are changes in this pull-request covered by:
- unit tests? N
- integration tests? N
